accio-tests/src/test/java/io/accio/testing/RequireAccioServer.java

```diff
@@ -83,6 +87,7 @@ public abstract class RequireAccioServer
     private static final JsonCodec<ConfigManager.ConfigEntry> CONFIG_ENTRY_JSON_CODEC = jsonCodec(ConfigManager.ConfigEntry.class);
     private static final JsonCodec<List<ConfigManager.ConfigEntry>> CONFIG_ENTRY_LIST_CODEC = listJsonCodec(ConfigManager.ConfigEntry.class);
     private static final JsonCodec<PreviewOutputDto> PREVIEW_OUTPUT_DTO_CODEC = jsonCodec(PreviewOutputDto.class);
+    private static final JsonCodec<DuckDBQueryDto> DUCK_DB_QUERY_DTO_JSON_CODEC = jsonCodec(DuckDBQueryDto.class);
```

Comment:
```suggestion
    private static final JsonCodec<DuckDBQueryDto> DUCKDB_QUERY_DTO_JSON_CODEC = jsonCodec(DuckDBQueryDto.class);
```
We use `DUCKDB` mostly.